rero_ils/modules/item_types/jsonschemas/item_types/item_type-v0.0.1.json

```
@@ -90,6 +94,97 @@
         }
       }
     },
+    "disable_circulation": {
+      "type": "boolean",
+      "title": "Mark the items of this category as not available",
+      "description": "If checked, none circulation operation can be performed on items related to this category.",
```

```suggestion
      "description": "If enabled, all items of this circulation category will be displayed as not available, with the red availability symbol.",
```

Note: I prefer "If enabled..." because it is already in use in the location editor.

---

👍 done